static/sass/_patterns_media-object.scss

```diff
@@ -7,5 +7,9 @@
     &__details {
       flex-grow: 1;
     }
+
+    &__meta-list-item {
```

Comment:
Maybe something we should report to Vanilla. Having only meta data without icons looks misaligned.